participants/david_leitner.json

```
@@ -0,0 +1,21 @@
+{
+    "name": "David Leitner",
+    "company": "SQUER Solutions",
+    "when": {
+        "friday": true,
+        "saturday": false
+    },
+    "iCanTakeNotesDuringSessions": false,
+    "tags": ["React", "Svelte", "Solid", "Next", "GraphQL"],
+    "vegan": false,
+    "vegetarian": false,
+    "whatIsMyConnectionToJavascript": "Engineer",
+    "whatCanIContribute": "I talk about the state of MicroFrontends",
+    "tShirt": {
+        "size": "L",
+        "type": "regular"
+    },
+    "twitter": "@duffleit",
```

We're using the Twitter handle in the link to Twitter, so we're using this without the `@` sign

```suggestion
    "twitter": "duffleit",
```